metrics/buckets.go

```diff
@@ -27,15 +27,15 @@ var NonNegativeBuckets = []float64{
 	1000000, 2000000, 5000000,
 	10000000, 20000000, 50000000,
 	100000000, 200000000, 500000000,
-	1000000000, 2000000000, 5000000000,
-	10000000000, 20000000000, 50000000000,
-	100000000000, 200000000000, 500000000000,
-	1000000000000, 2000000000000, 5000000000000,
-	10000000000000, 20000000000000, 50000000000000,
-	100000000000000, 200000000000000, 500000000000000,
-	1000000000000000, 2000000000000000, 5000000000000000,
-	10000000000000000, 20000000000000000, 50000000000000000,
-	100000000000000000, 200000000000000000, 500000000000000000,
-	1000000000000000000, 2000000000000000000, 5000000000000000000,
-	10000000000000000000, 20000000000000000000, 50000000000000000000, // i.e., 5e19
+	1000000000, 2000000000, 5000000000, // i.e., 5e9
```

Comment:
Can we leave `NonNegativeBuckets` as-is, since it's not just used by the auto-generated metrics but is public to the user?

Instead, let's create a different internal variable in `weaver/http.go` and use that for the auto-generated metrics.

Reply:
Good point.

weaver/http is in package weaver, so given that I have to use custom buckets in 3 different places, I decided to  define the buckets locally in each file.

Regarding `NonNegativeBuckets` - I feel that the number of buckets is really large for any customer, and they will keep complaining if this is the only option we provide. They can define their custom buckets when they create histograms though. I am not sure if we should provide any `NonNegativeBuckets` at all (this way the user will always have to define their buckets), or if we should provide a few options with different number of buckets.

Reply:
I'd be in favor of removing it, if it's not used by any other code.

Reply:
Sounds good. I will remove it in a future PR.